resources/admin/calendar.js

```diff
@@ -35,12 +35,16 @@ const tooltipContent = ({ _instance, _def }, hideLocation) => {
     const { start, end } = _instance.range;
     const { title, extendedProps } = _def;
     const contactPerson = extendedProps.contact_person
-    const { assignment_count, experiment_url, session_url, max_participants, min_participants, overbook } = extendedProps;
-    const counterHtml = `<p><strong>Participants: ${assignment_count} / ${max_participants}</strong><br>Overbook: ${overbook}<br>Min. participants: ${min_participants}</p>`
-    const linksHtml = `<p>
-        <a href="${experiment_url}">Experiment details</a><br>
-        <a href="${session_url}">Session details</a>
-    </p>`
+    const { assignment_count, max_participants, min_participants, overbook, links } = extendedProps;
+    const { show_experiment, show_session, show_location_session, experiment, session, location_session } = links;
+    const counterHtml = `<p><strong>Participants: ${assignment_count} / ${max_participants}</strong><br>Overbook: ${overbook}<br>Min. participants: ${min_participants}</p>`;
+    const sessionLink = show_session ? `<a href="${session}">Session details</a>` : '';
+    const locationSessionLink = show_location_session ? `<a href="${location_session}">Session details</a>` : '';
+    const experimentLink = show_experiment ? `<a href="${experiment}">Experiment details</a>` : '';
+    var linkList = [];
+    show_session ? linkList.push(sessionLink) : show_location_session ? linkList.push(locationSessionLink) : null;
```

Comment:
Can we get rid of those nested tenary operators? Either by updating the link type as described or by using if and else if? 

E.g.
```
    if (show_session) {
        linkList.push(sessionLink)
    } else if (show_location_session) {
        linkList.push(locationSessionLink)
    }
```